spec/decorators/note_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe NoteDecorator do
+  subject(:decorated) { note.decorate }
+
+  describe ".created_at" do
+    let(:note) { build(:note, created_at: Date.new(1970, 0o1, 0o1)) }
```

Comment:
I didn't understand why we need to use `o` in `0o1`. Can you clarify?

Reply:
It is a mistake. I used `01, 01` for date to be more readable and linter corrected it to `0o1, 0o1`. I'll fix this.